.travis.yml

```diff
@@ -37,20 +37,26 @@ matrix:
       language: minimal
       install:
         - docker-compose build
+        - pip install codecov --user
```

Comment:
I don't believe `--user` is necessary'. Am I correct?

Reply:
Locally it wasn't needed. But when I tried my build, it failed stating that I need to put --user for providing privileges to installation.